src/components/dateTimePicker/index.tsx

```diff
@@ -233,8 +243,15 @@ const DateTimePicker = forwardRef((props: DateTimePickerPropsInternal, ref: Forw
           iconStyle={{tintColor: Colors.$iconDefault}}
           onPress={toggleExpandableOverlay}
           testID={`${testID}.cancel`}
+          {...cancelButtonProps}
+        />
+        <Button
+          link
+          iconSource={Assets.icons.check}
+          onPress={onDonePressed}
+          testID={`${testID}.done`}
+          {...confirmButtonProps}
```

Comment:
make sure to pass our internal onPress after the spread otherwise we might lost the functionality